src/hotspot/share/gc/shenandoah/shenandoahBufferWithSATBRememberedSet.inline.hpp

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Missing copyright header too.

Reply:
And header #ifndef-#define-#endif ;-)

Reply:
As discussed in email message, this file has been removed from the pull request.